.github/workflows/compare-chromium-versions.yml

```diff
@@ -0,0 +1,41 @@
+name: Compare Chromium versions
+
+on:
+  pull_request:
+    types: [opened, reopened, synchronize]
+
+permissions:
+  pull-requests: write
+
+jobs:
+  compare:
+    runs-on: ubuntu-latest
+    env:
+      GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      PR_NUMBER: ${{ github.event.number }}
+    steps:
+      - run: |
+          shopt -s inherit_errexit
+          set -xeEo pipefail
```

Comment:
sadly, the error/status message is not displayed in the PR checks area, so we had to open the job details to see what happened. The current output does not look user-friendly, it's hard to distinguish the line with the important information. Can we disable the verbose command output? Or move the result reporting into a separate step?

<img width="713" alt="image" src="https://github.com/user-attachments/assets/18842501-b8e6-4da0-9575-6b854ccbfe7c">

Reply:
> When builds fail due to a version mismatch, the information about what happened is included in the Slack notification.

that's cool, but having a clear error when you click "Details" is also important. If you want to keep the debug output, then splitting the workflow into two steps will allow you to keep the verbose output for debugging, and at the same time will make it easy for every developer to see what's wrong.

> If we really want to go beyond that, we could set a custom status text for the CI jobs (which we know how to do), or look into how that can be done for this check (not sure atm).

We can use [Status API](https://docs.github.com/en/rest/commits/statuses?apiVersion=2022-11-28#create-a-commit-status), but this will create additional check in the checks list. If we go this way, we can make the workflow always succeed (and rename it to something like "PR checks"), but make it create a failed/successful chromium version check with the exact description right in the PR checks area.